synapse/rest/client/v2_alpha/devices.py

```diff
@@ -45,6 +45,52 @@ def on_GET(self, request):
         )
         defer.returnValue((200, {"devices": devices}))
 
+class DeleteDevicesRestServlet(servlet.RestServlet):
+    PATTERNS = client_v2_patterns("/delete_devices", releases=[], v2_alpha=False)
```

Comment:
Could you add a docstring to say what this and what form the API roughly takes, e.g.:

> API for bulk deletion of devices. Accepts a json object with a `devices` key which lists the device_ids to delete. Requires user interactive auth.